RZUtils/Components/RZSingleChildContainerViewController/RZSingleChildContainerViewController.m

```diff
@@ -126,7 +126,7 @@ - (UIViewController *)childViewControllerForStatusBarHidden
     return self.currentContentViewController;
 }
 
-- (NSUInteger)supportedInterfaceOrientations
```

Comment:
Yes, it did on the Travis build with Xcode 6.1. If you want the conditional compilation, I'll add it.